Latest ~1.6.x version of json-schema-to-ts causes

Starting from version 1.6.0, `json-schema-to-ts` causes compilation to fail.

```sh
> typebridge@0.4.0 type /Users/Frederic/Sites/typebridge
> tsc --noEmit

src/Bus.ts:74:13 - error TS2589: Type instantiation is excessively deep and possibly infinite.

74     events: Event<string, Record<string, unknown>>[],
```

--- a/.npmrc
+++ b/.npmrc
@@ -1,0 +1,2 @@
+engine-strict=true
+package-lock=false

--- a/.prettierrc
+++ b/.prettierrc
@@ -1,6 +1,5 @@
 {
   "singleQuote": true,
   "trailingComma": "all",
-  "jsxBracketSameLine": false,
   "bracketSpacing": true
 }

--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
   },
   "dependencies": {
     "ajv": "^6.12.6",
-    "json-schema-to-ts": "^1.4.3"
+    "json-schema-to-ts": "1.5.0"
   },
   "devDependencies": {
     "@middy/core": "^1.4.0",

--- a/src/Bus.ts
+++ b/src/Bus.ts
@@ -70,9 +70,10 @@ export class Bus {
     );
   }
 
-  computePattern(
-    events: Event<string, Record<string, unknown>>[],
-  ): { source?: string[]; 'detail-type'?: string[] } {
+  computePattern(events: Event<string, Record<string, unknown>>[]): {
+    source?: string[];
+    'detail-type'?: string[];
+  } {
     const pattern = {};
     const areAllEventSourcesEqual = events.every(
       (event) => event.source === events[0].source,

